workbench/electron-builder-config.js

```diff
@@ -19,9 +19,16 @@ const APP_ID = `NaturalCapitalProject.Invest.Workbench.${investVersion}`;
 const PRODUCT_NAME = `InVEST ${investVersion} Workbench`;
 const ARTIFACT_NAME = `invest_${investVersion}_workbench_${OS}_${ARCH}.${EXT}`;
 
+// this version appears as a footer in the NSIS installer & uninstaller.
+// It includes package.json version by default, but that is meaningless for us.
+// We can override that, but also must maintain semver compliance, so
+// always trim to this format.
+const installerVersion = investVersion.match(/[0-9]+\.[0-9]+\.[0-9]+/)[0];
+
```

Comment:
Interesting!  It does look like [SemVer 2.0 does support build metadata in the version string](https://semver.org/#spec-item-10), but not in quite the format that we've had it.  Would the extra version metadata be useful to us?

Reply:
Oh interesting. So the place that this appears is actually at the very end of a string that already includes the fuller version string with build metadata.
![Capture](https://user-images.githubusercontent.com/4071255/224784475-50a68781-9879-44ea-b6e9-07ea04225a5c.PNG)

I would have replaced this with nothing if I could have!

Reply:
Basically, we construct a custom `productName` and `artifactName` using invest's string. But `electron-builder` still always appends the version it finds in `package.json`. And `package.json` requires a `version` property. We have a fairly bespoke project here, with node + python.

Reply:
Huh, I wouldn't have expected both versions in there!  In that case, let's just go with what you have here, that sounds great.  Thanks!